.github/workflows/tomlcheck.yaml

```diff
@@ -0,0 +1,48 @@
+# SPDX-License-Identifier: Apache-2.0
+
+name: TOML check
+
+on:
+  push:
+    branches:
+      - "main"
+      - "release-**"
+    paths:
+      - 'pyproject.toml'
+      - '.github/workflows/tomlcheck.yaml' # This workflow file
+  pull_request:
+    branches:
+      - "main"
+      - "release-**"
+    paths:
+      - 'pyproject.toml'
+      - '.github/workflows/tomlcheck.yaml' # This workflow file
+
+env:
+  LC_ALL: en_US.UTF-8
+
+defaults:
+  run:
+    shell: bash
+
+permissions:
+  contents: read
+
+jobs:
+  tomlcheck:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: lint pyproject.toml
+        run: |
+          make toml-lint
```

Comment:
Why not add this to the existing lint workflow? The number of workflows seems to be growing...

Reply:
I did not add it to the lint workflow to have a small and isolated workflow. If only the `pyproject.toml` is changed then, only the TOML linter will run.

Reply:
Each new workflow complicates the mergify config as you demonstrate by having to add more mergify config.

Reply:
Makes sense, I applied your suggestion and incorporated the `pyproject.toml` lint/fmt to the lint action workflow. Thanks!